Lesson17/feyza/script.js

```diff
@@ -0,0 +1,133 @@
+// For loop
+const myArray = ['element1', 'element2', 'element3'];
+for (let i = 0; i < myArray.length; i++) {
+	console.log(myArray[i], 'i is:', i);
+}
+// while
+// do while
+// Array.prototype.map function - we'll learn it later
+
+// 1. Write a loop that prints out the numbers from 1 to 10 (not included).
+for (let i = 1; i < 10; i++) {
+	console.log(i);
+}
+console.log('- - - - - - - - - - - - - - - - - ');
+// 2. Write a loop that prints out the odd numbers from 1 to 20.
+// 1,3,5,7...
+
+for (let i = 1; i < 20; i += 2) {
+	console.log(i);
+}
+
+console.log('- - - - - - - - - - - - - - - - - ');
+// 3. Write a loop that prints out the even numbers from 1 to 20.
+// 2, 4, 6...
+for (let i = 0; i < 20; i += 2) {
```

Comment:
This loop should start with number 2, not 0 so define `i` like this instead:
`let i = 2;`.
Also, 20 should be included so that you need to use 'i<=20' isntead.